### ISSUE TITLE
USER ISSUE: Learning wrong role can screw up your game right from the start

### ISSUE THREAD

**Issue by Conchipedia** [StrangeLoopGames/EcoIssues#3384]
**Version:** 0.6.4.0 alpha staging-d2f49058

Playing this game as a new player following Tutorial

I chose Farmer (Digging) when it said choose a role because that was 0 points, now if i try and follow the rest of the tutorial i cannot, as to make a stockpile you will need an axe and this is 38 points and at a Skill rate of 22.7 (just eating tomato's) this will take me two days of NOT playing and i do not see a New player coming back to the game if they have to wait that long, that is if they even realise that they now cannot do anything but just gather food till backpack is full because they cannot even make a chest to store this.

and you cannot UNLEARN anything (if it gives you an item) to make another choice of a role

First Priority Must be to Build a House/Room without an Axe (38 points to unlock) you cannot do that, and then crafting will be impossible because you then need a pickaxe (38 points to unlock) and that cannot be made because you need stone ingredients for the Recipe and that requires a Pickaxe.

Perhaps the Choose a role is not a good idea to place to early, their First priority would be to unlock    **"Basic Crafting"**

So why not under Basic Crafting that gives us a workbench and a hammer (still the wrong type as that requires iron ingot to repair) Give the player this:

**Workbench, Stone Hammer, Wooden Shovel, Stone Axe and a Stone Pickaxe (Hoe and Scythe can be crafted later when needed)**

this will then at least give the Player the **Basic Tools** for crafting and Clearing space, then they can choose which role they want to do After they have at least managed to Build a house and storage.

![snap1050](https://user-images.githubusercontent.com/29383891/34907082-0e22c714-f871-11e7-874f-fa2f2d7a29ac.png)

To reproduce this: Start as a Fresh Player do NOT give yourself any skills just learn Digging (or anything else that doesn't give you an axe) as a role.

**Comment by Conchipedia** [comment]
For Now (as i cannot do anything on Marmot) i will go and play my Normal Eco, and unless Marmot gets wiped i will come back occasionally and check my SP to see if i can unlock the Axe and carry on with testing.

**Comment by Arend90** [comment]
this is the 3rd post you write about this, you take 3 specialzations your self 🗡  find 5 friends instead of wanting to have 10 specializations 👎  lonewolf - 💯

**Comment by Conchipedia** [comment]
i made a mistake and took Farming,  (i was following tutorial about choosing a role, it didn't say i MUST make sure i have an axe first) now i cannot do anything until i get 38 points to then get an axe to then continue with the game following the tutorial as my next task after choosing a role is : craft a stockpile which requires me to chop down a tree for the ingredients.

**Comment by Arend90** [comment]
We talked about a /removedeleteplayer maybe thats not a bad idea.
really think a lot of players have to start over as they not use to this, everybody took all the tools at start

**Comment by Lussian-A** [comment]
Pardon the intrusion, but I'd like to toss in my 2 cents here as well.

I believe that the start of the game needs a redesign and has very little with actually being a lonewolf.
A lot of the basic skills don't have much incentive ... err that entices you to take them, aside from gettin the tools to get started. Which is why they worked when they were readily available in the survivor tree for a low skill point cost. 

It would probably be better to have people start with whatever tools you deem basic and instead add something that makes people wanna spec into logging, digging and such. As logging had before, with the bonus log per pick. Same as the various gathering skills have, but others lack, as digging and mining. As reduced cal cost is ... not that great when food and better tools become available.

Well some of my 2 cents anyways. ta ta

**Comment by Conchipedia** [comment]
that's what i was thinking give us the basic tools but NO BONUSES for using them, we still have to learn that to get extra things

**Comment by Subtra** [comment]
I got an issue with the issue being a lone Wolf, the game isnt designed for you to be a lone wolf to be a master of everything. The Game needs you to have some Time to spare, if you choose everything at beginning of the skill tree, you can first train up to the repair Station after your choosen profession, but the Game want you to work together with others, There are enoug lone wolf games. You can make your own server and configure the game so you dont wait forever for your skill points, but thats only if you want to play alone on this world. So think about it.

**Comment by MarjieAVolk** [comment]
@Conchipedia about the unlearning, I am going to change how that works so that you can unlearn all those basic roles.  Maybe that will help with some of your issues.

We definitely need to review the tutorial content more before beta, as it's already getting out of date with our new changes.  Thanks all for your feedback, as this is a pretty big change that's going to need more balancing, it's good to hear from players about it.

**Comment by Conchipedia** [comment]
having the ability to unlearn at the start when u realise about a mistake is good, especially the ones that gives an item (just program it to have a check that if you have gotten an item and unlearn it already knows you have received this item and will not give you another one) should stop people cheat unlearning for free stuff

**Comment by craig-jong** [comment]
I'm glad you guys are having a discussion about this topic. As Marjie already said there's going to be some balance changes for sure.

Something that is extremely important is that we shouldn't let players get into a state where its impossible for them to do anything meaningful except wait. We have recipes for all the starting tools on the workbench, but with no way to gather some of the materials without tools it doesn't really help that much.

A good tutorial can mitigate this, but I'm hoping that eventually we can add in a few "hand-pickable" resource points for stone and wood that, while accessible without the proper tools, give significantly less resources.

**Comment by Conchipedia** [comment]
After playing for a bit, i have realized that until i learn woodworking 1 i cannot repair a Wooden Shovel or Wooden Hoe, and also without Stoneworking 1 all Stone tools cannot be repaired either, so at the moment until my friends come back i am just making NEW tools and shoving the broken ones in a chest.

I had only unlocked Wooden Shovel by choosing farming, i was lucky that we decided to test on Marmot and they made me a Stone Pickaxe and Stone Axe, at least i can collect logs and Stone place them in a Stockpile and then just make More tools.

This is one of the reasons for unlocking the basic 4 Tools (Axe, Hammer, Pickaxe and Shovel) at least the player can have a play around making storage and geting used to doing stuff while they are waiting for their SP to go up (they can even craft a Hoe and plant the seeds they have harvested when gathering)

This is not in my opinion Lonewolfing as the Player Cannot unlock things until they have enough SP (which can take ages) But should encourage people to work together more and decide which person then takes what role.
